lib/foreman/renderer/configuration.rb

```diff
@@ -37,7 +37,8 @@ class Configuration
         :number_to_percentage,
         :number_with_delimiter,
         :number_with_precision,
-        :number_to_human_size
+        :number_to_human_size,
+        :gem_version_compare
```

Comment:
I thought we had trailing comma's but looks like we don't here. @mmoll is that a missing rubocop rule?

Reply:
Yes, that should be `Style/TrailingCommaInArrayLiteral`... I'd need to have a deeper look to find out, if that's a Rubocop bug.